approval_test.go

```diff
@@ -112,11 +114,43 @@ func TestApprovalFromComments(t *testing.T) {
 			approvers:      []string{login1, login2},
 			expectedStatus: approvalStatusDenied,
 		},
+		{
+			name: "multi_approver_minimum_one_approval",
+			comments: []*github.IssueComment{
+				{
+					User: &github.User{Login: &login1},
+					Body: &bodyPending,
+				},
+				{
+					User: &github.User{Login: &login2},
+					Body: &bodyApproved,
+				},
+			},
+			approvers:        []string{login1, login2},
+			expectedStatus:   approvalStatusApproved,
+			minimumApprovals: 1,
+		},
+		{
+			name: "multi_approver_minimum_two_approvals",
+			comments: []*github.IssueComment{
+				{
+					User: &github.User{Login: &login1},
+					Body: &bodyApproved,
+				},
+				{
+					User: &github.User{Login: &login2},
+					Body: &bodyApproved,
+				},
+			},
+			approvers:        []string{login1, login2, login3},
+			expectedStatus:   approvalStatusApproved,
+			minimumApprovals: 2,
+		},
```

Comment:
Can you add a test case with multiple approvers and the number of comment approvals is less than `minimumApprovals`? I'd like to make sure we cover the case where we have *some* approvals, but not enough to meet minimum approvals. So perhaps 3 approvers, `minimumApprovals` set to 2, and only 1 approved message.

Reply:
I have added the test case `multi_approver_approvals_less_than_minimum` to cover this